events/oio_events_queue_beanstalkd.c

```diff
@@ -478,3 +484,19 @@ _q_is_stalled (struct oio_events_queue_s *self)
 		return FALSE;
 	return ((guint)l) >= oio_events_common_max_pending;
 }
+
+static gint64
+_q_get_health(struct oio_events_queue_s *self)
+{
+	struct _queue_BEANSTALKD_s *q = (struct _queue_BEANSTALKD_s*) self;
+	EXTRA_ASSERT(q != NULL && q->vtable == &vtable_BEANSTALKD);
+
+	gint64 limit = G_MAXINT64;
+	if (oio_events_beanstalkd_check_level_deny > 0)
+		limit = oio_events_beanstalkd_check_level_deny;
+	else if (oio_events_beanstalkd_check_level_alert > 0)
```

Comment:
This is incorrect. That configuration variable can only be used for warnings to be displayed and must not trigger the zeroing of a service's score.

Reply:
This does not trigger the zeroing of the service.